beginner/building-first-pipeline/python/src/README.md

```diff
@@ -0,0 +1,50 @@
+# Tutorial pipeline for Python
+
+This pipeline is based on the Python tutorial documents available at:
+
+  https://tutorials.datajoint.io/
+
+with data available from:
+
+  https://tutorials.datajoint.io/_downloads/python-data.zip
+
+## Tutorial Schema
+![Entity-Relationship Diagram](first-pipeline-python.png)
+
+## Obtain credentials
+
+If you need a database account to try these examples, you can get a free
+tutorial account by subscribing through https://datajoint.io.
+
+Before you start working with the pipeline, please obtain the following
+credentials:
+
+* host address
+* user name 
+* password
+
+# Setup
+
+The instructions for downloading the DataJoint library are available here:
+
+http://docs.datajoint.io/setup/Install-and-connect.html
+
+Be sure to set your PYTHONPATH variable accordingly so that the
```

Comment:
Would be better to have a separate instructions on how to modify the `PYTHONPATH` or actually ignore the discussion of this all together.